src/solver.rs

```diff
@@ -0,0 +1,134 @@
+//! Solvers for detecting known script kinds.
+
+use bounded_vec::BoundedVec;
+
+use crate::{
+    external::pubkey::PubKey,
```

Comment:
Found a bug while updating https://github.com/zcash/librustzcash/pull/1721: this is now gated off, but we need it for its constants (which @sellout made associated constants somewhere in #174).

Reply:
Opened #266 for updating CI to catch this kind of issue going forward.